### DISCUSSION TITLE
Test Suite issues

### DISCUSSION THREAD

**Discussion by carsonwright** [adonisjs/core#1915] [Help]
I followed this tutorial, but I had issues using adonis assembler build register.
https://preview.adonisjs.com/blog/running-tests-in-adonisjs-v5

```
node -r @adonisjs/assembler/build/register japaFile.ts

internal/modules/cjs/loader.js:834
  throw err;
  ^

Error: Cannot find module '@adonisjs/assembler/build/register'
Require stack:
- internal/preload
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:831:15)
    at Function.Module._load (internal/modules/cjs/loader.js:687:27)
    at Module.require (internal/modules/cjs/loader.js:903:19)
    at Module._preloadModules (internal/modules/cjs/loader.js:1155:12)
    at loadPreloadModules (internal/bootstrap/pre_execution.js:446:5)
    at prepareMainThreadExecution (internal/bootstrap/pre_execution.js:74:3)
    at internal/main/run_main_module.js:7:1 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ 'internal/preload' ]
}
```

**Comment by carsonwright** [comment]
<img width="764" alt="Screen Shot 2020-11-16 at 10 08 26 PM" src="https://user-images.githubusercontent.com/1863555/99341727-6f089a80-2858-11eb-84b6-591f5695d7cf.png">

**Comment by RomainLanz** [comment]
Hey @carsonwright! 👋 

Which version of `@adonisjs/assembler` are you using?

**Reply by carsonwright** [reply]
I don't know actually, how do I tell that? I didn't set any of it up, I let the system build its own version 5 setup and didn't get involved. Is there a specific node module I can look up for that?

**Reply by thetutlage** [reply]
Check it inside your package.json file?

**Reply by carsonwright** [reply]
"@adonisjs/assembler": "^2.0.0",

you're right the creation of the project must have generated this dependency. Supposedly it is for 5.x.x

**Reply by carsonwright** [reply]
Sorry I didn't add that dependency so I didn't even realize it was there. Was pretty surprised to find it.

**Reply by carsonwright** [reply]
@thetutlage

**Reply by thetutlage** [reply]
The dependency was added when you created a new project. You have to make sure you are running the latest versions of all dependencies.
